services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx

```diff
@@ -22,6 +22,25 @@ import { Tooltip } from '@material-ui/core';
 import { useNavigate } from 'react-router-dom';
 import * as React from 'react';
 
+// number of releases on home. based on design
+const numberOfDisplayedReleasesOnHome = 6;
+
+const getAllDisplayedReleases = (deployedReleases: number[], allReleases: number[]): number[] => {
+    // number of remaining releases to get from history
+    const numOfTrailingReleases = numberOfDisplayedReleasesOnHome - deployedReleases.length;
+    // last deployed release e.g. Prod
+    const oldestDeployedRelease = deployedReleases[deployedReleases.length - 1];
```

Comment:
this will crash if `deployedReleases` is empty, which isn't likely, but can happen.

Reply:
good catch! thanks :)